Parse/Internal/User/PFUserPrivate.h

```diff
@@ -51,20 +51,17 @@ extern NSString *const PFUserCurrentUserKeychainItemName;
 @end
 
 // Private Properties
-@interface PFUser () {
-    BOOL isCurrentUser;
-    NSMutableDictionary *authData;
-    NSMutableSet *linkedServiceNames;
-    BOOL isLazy;
-}
-
-// This earmarks the user as being an "identity" user. This will make saves write through
-// to the currentUser singleton and disk object
-@property (nonatomic, assign) BOOL isCurrentUser;
-
-@property (nonatomic, strong, readonly) NSMutableDictionary *authData;
-@property (nonatomic, strong, readonly) NSMutableSet *linkedServiceNames;
-@property (nonatomic, assign) BOOL isLazy;
+@interface PFUser ()
+
+@property (nonatomic, strong, readonly) NSMutableDictionary<NSString *, id> *authData;
+@property (nonatomic, strong, readonly) NSMutableSet<NSString *> *linkedServiceNames;
+
+/**
+ This earmarks the user as being an "identity" user.
+ This will make saves write through to the currentUser singleton and disk object
+ */
+@property (nonatomic, assign, getter=isCurrent, setter=setIsCurrent:) BOOL current;
```

Comment:
Because `PFUser` could be subclassed, we should try to pick property names that are less likely to conflict (remember `state`, anyone?). I fear that `lazy` and `current` may already be in use by some applications, and I'd hate to cause incompatibilities for existing code.